src/ifcconvert/ColladaSerializer.h

```diff
@@ -150,7 +150,9 @@ class ColladaSerializer : public GeometrySerializer
 	public:
 		ColladaExporter(const std::string& scene_name, const std::string& fn, ColladaSerializer *_serializer)
 			: filename(fn.c_str())
-			, stream(filename)
+            /// @todo We could use the double precision feature here, but it seems to bloat the output file quite a bit
```

Comment:
Yeah, looking the numbers a bit more closely, if `doublePrecision = true` is used, the values in DAE seem to be closer to what they are in IFC.